snippets/css-snippets.cson

```diff
@@ -1,689 +1,707 @@
+    'body': 'font-weight: normal'
   'height':
     'prefix': 'hei'
-    'body': 'height: ${1:${2:1}${3:px}}$0'
+    'body': 'height: ${1:${2:1}${3:px}}'
   'height: auto':
     'prefix': 'heia'
-    'body': 'height: auto$0'
+    'body': 'height: auto'
   'list-style':
     'prefix': 'lis'
-    'body': 'list-style: ${1:square} ${2:inside} ${4:url(\'${3:list.png}\')}$0'
+    'body': 'list-style: ${1:square} ${2:inside} ${4:url(\'${3:list.png}\')}'
   'list-style-type':
     'prefix': 'list'
-    'body': 'list-style-type: ${1:circle}$0'
+    'body': 'list-style-type: ${1:circle}'
   'list-style-type: circle':
     'prefix': 'listc'
-    'body': 'list-style-type: circle$0'
+    'body': 'list-style-type: circle'
   'list-style-type: disc':
     'prefix': 'listd'
-    'body': 'list-style-type: disc$0'
+    'body': 'list-style-type: disc'
   'list-style-type: lower-roman':
     'prefix': 'listlr'
-    'body': 'list-style-type: lower-roman$0'
+    'body': 'list-style-type: lower-roman'
   'list-style-type: none':
     'prefix': 'listn'
-    'body': 'list-style-type: none$0'
+    'body': 'list-style-type: none'
   'list-style-type: square':
     'prefix': 'lists'
-    'body': 'list-style-type: square$0'
+    'body': 'list-style-type: square'
   'list-style-type: upper-roman':
     'prefix': 'listur'
-    'body': 'list-style-type: upper-roman$0'
+    'body': 'list-style-type: upper-roman'
   'left':
     'prefix': 'left'
-    'body': 'left: ${1:0}$0'
+    'body': 'left: ${1:0}'
   'left (shorthand)':
     'prefix': 'lef'
-    'body': 'left: ${1:0}$0'
+    'body': 'left: ${1:0}'
   'line-height':
     'prefix': 'lh'
-    'body': 'line-height: ${1:${2:1}${3:em}}$0'
+    'body': 'line-height: ${1:${2:1}${3:em}}'
   'letter-spacing':
     'prefix': 'ls'
-    'body': 'letter-spacing: ${1:${2:1}${3:px}}$0'
+    'body': 'letter-spacing: ${1:${2:1}${3:px}}'
   'letter-spacing: normal':
     'prefix': 'lsn'
-    'body': 'letter-spacing: normal$0'
+    'body': 'letter-spacing: normal'
   'margin':
     'prefix': 'mar'
-    'body': 'margin: ${1:${2:1}${3:px}}$0'
+    'body': 'margin: ${1:${2:1}${3:px}}'
   'margin-bottom':
     'prefix': 'marb'
-    'body': 'margin-bottom: ${1:${2:1}${3:px}}$0'
+    'body': 'margin-bottom: ${1:${2:1}${3:px}}'
   'margin-left':
     'prefix': 'marl'
-    'body': 'margin-left: ${1:${2:1}${3:px}}$0'
+    'body': 'margin-left: ${1:${2:1}${3:px}}'
   'margin-right':
     'prefix': 'marr'
-    'body': 'margin-right: ${1:${2:1}${3:px}}$0'
+    'body': 'margin-right: ${1:${2:1}${3:px}}'
   'margin-top':
     'prefix': 'mart'
-    'body': 'margin-top: ${1:${2:1}${3:px}}$0'
+    'body': 'margin-top: ${1:${2:1}${3:px}}'
   'min-height':
     'prefix': 'mh'
-    'body': 'min-height: ${1:${2:1}${3:px}}$0'
+    'body': 'min-height: ${1:${2:1}${3:px}}'
   'min-width':
     'prefix': 'mw'
-    'body': 'min-width: ${1:${2:1}${3:px}}$0'
+    'body': 'min-width: ${1:${2:1}${3:px}}'
+  'max-height':
+    'prefix': 'maxh'
```

Comment:
As above, this will be fine, I just did the prefixes based on the first three letters you'd type to get the snippet'd code (e.g. _box_, _max_ etc) - Do you have any objections to aliases for some snippets?